test/unit/mdc-drawer/mdc-persistent-drawer.js

```diff
@@ -0,0 +1,284 @@
+});
+
+test('adapter#getDrawerWidth returns the width of the drawer element', () => {
+  const {root, component} = setupTest();
+  const drawer = root.querySelector(strings.DRAWER_SELECTOR);
+  assert.equal(component.getDefaultFoundation().adapter_.getDrawerWidth(), drawer.offsetWidth);
+});
+
+test('adapter#setTranslateX sets the correct transform on the drawer', () => {
+  const {root, component} = setupTest();
+  const drawer = root.querySelector(strings.DRAWER_SELECTOR);
+  component.getDefaultFoundation().adapter_.setTranslateX(42);
+  assert.equal(drawer.style.getPropertyValue(getTransformPropertyName()), 'translateX(42px)');
+});
+
+test('adapter#setTranslateX sets translateX to null when given the null value', () => {
+  const {root, component} = setupTest();
+  const drawer = root.querySelector(strings.DRAWER_SELECTOR);
+  component.getDefaultFoundation().adapter_.setTranslateX(null);
+  const transformPropertyName = getTransformPropertyName();
+  const nonStyledElement = document.createElement('div');
+  assert.equal(
+    drawer.style.getPropertyValue(transformPropertyName), nonStyledElement.style.getPropertyValue(transformPropertyName)
+  );
+});
+
+test('adapter#updateCssVariable sets custom property on root', () => {
+  const {root, component} = setupTest();
+  component.getDefaultFoundation().adapter_.updateCssVariable('0');
+  if (supportsCssCustomProperties()) {
+    assert.equal(root.style.getPropertyValue(strings.OPACITY_VAR_NAME), '0');
+  }
+});
+
+test('adapter#getFocusableElements returns all the focusable elements in the drawer', () => {
+  const root = bel`
+    <aside class="mdc-persistent-drawer">
+      <nav class="mdc-persistent-drawer__drawer">
+        <div class="mdc-persistent-drawer__toolbar-spacer"></div>
+        <button></button>
+        <a href="foo"></a>
+        <div></div>
+        <div tabindex="0"></div>
+      </nav>
+    </aside>
+  `;
+  const component = new MDCPersistentDrawer(root);
+  assert.equal(component.getDefaultFoundation().adapter_.getFocusableElements().length, 3);
+});
+
+test('adapter#makeElementUntabbable sets a tab index of -1 on the element', () => {
+  const root = bel`
+    <aside class="mdc-persistent-drawer mdc-persistent-drawer--open">
+      <nav class="mdc-persistent-drawer__drawer">
+        <a id="foo" href="foo"></a>
+      </nav>
+    </aside>
+  `;
+  const component = new MDCPersistentDrawer(root);
+  const el = root.querySelector('#foo');
+  component.getDefaultFoundation().adapter_.makeElementUntabbable(el);
+  assert.equal(el.getAttribute('tabindex'), '-1');
+});
+
+test('adapter#isRtl returns true for RTL documents', () => {
+  const root = bel`
+    <aside dir="rtl" class="mdc-persistent-drawer">
+      <nav class="mdc-persistent-drawer__drawer">
+      </nav>
+    </aside>
+  `;
+  document.body.appendChild(root);
+  const component = new MDCPersistentDrawer(root);
+  assert.isOk(component.getDefaultFoundation().adapter_.isRtl());
```

Comment:
you'll want to make sure to remove the root element from `document.body` before the test ends, here and below.